docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -34,6 +34,38 @@ image::images/cases-ui-connector.png[Shows the page for creating connectors]
 - {kibana-ref}/swimlane-action-type.html[{swimlane} connector]
 - {kibana-ref}/cases-webhook-action-type.html[{webhook-cm} connector]
 
+[float]
+[[mapped-case-fields]]
+=== Mapped case fields
+
+To represent an {es-sec} case in an external system, {es-sec} case fields are mapped to existing fields in {sn}, {jira}, {ibm-r}, and {swimlane}. Once fields are mapped, you can push updates to external systems and mapped fields are overwritten or appended. Retrieving data from external systems is not supported.
+
+|===
+
+| *Case field* | *Mapped field*
+
+| Title
+
+a| The case `Title` field is mapped to corresponding fields in external systems. Mapped field values are overwritten when you push updates.
+
+* *{sn}*: Mapped to the `Short description` field.
+* *{jira}*: Mapped to the `Summary` field.
+* *{ibm-r}*: Mapped to the `Name` field.
+* *{swimlane}*: Mapped to the `Description` field.
```

Comment:
The sentence above already says that the case field is mapped to an external field, so no need to repeat that in every bullet.
```suggestion
* *{sn}*: `Short description`
* *{jira}*: `Summary`
* *{ibm-r}*: `Name`
* *{swimlane}*: `Description`
```